work/worker.go

```diff
@@ -478,9 +476,9 @@ func (self *worker) commitNewWork() {
 
 	// TODO-Klaytn drop or missing tx
 	tstamp := tstart.Unix()
-	if self.nodetype == node.CONSENSUSNODE {
+	if self.nodetype == common.CONSENSUSNODE {
 		if parent.Time().Cmp(new(big.Int).SetInt64(tstamp)) >= 0 {
-			//if self.nodetype == node.ENDPOINTNODE {
+			//if self.nodetype == p2p.ENDPOINTNODE {
```

Comment:
Why did you modify the comment here?

Reply:
@nogang This is something caused by global pattern replacement or the feature of GoLand.